technology-overview.md

```diff
@@ -0,0 +1,84 @@
+# Technology overview
+
+[Agoric](https://agoric.com/about/) aims at enabling safe cooperation among mutually suspicious parties
+
+The way it does it is by making it easy to write secure smart contracts and making it possible to run these contracts on one or several blockchains. Agoric strategy to writing smart contracts is to write them in JavaScript using a framework called ERTP (Electronic Rights Transfer Protocol)
```

Comment:
"The way it does it is by" -> "It does this by"

I am conflicted about "easy". It is too strong. OTOH "easier" is too weak, as it would be consistent with only a mild quantitative improvement. Somewhere we have phrasing that dealt with this dilemma. Until we find it or come up with something, better to err towards claiming too little rather than too much.

The "easy" vs "possible" juxtaposition makes it seem like there's something hard about running these on several blockchains. For the user, the latter should be easy as our platform presents a chain/platform independent abstraction.

Reply:
> The "easy" vs "possible" juxtaposition makes it seem like there's something hard about running these on several blockchains. For the user, the latter should be easy as our platform presents a chain/platform independent abstraction.

My intention with this paragraph is to explain the contributions of Agoric when it comes to the smart contract authoring ecosystem. Of which i think there are two:
1) With Agoric work, it becomes easi**er** to write secure smart contracts
2) With Agoric, it comes possible to write cross-blockchain contracts (which i assume is currently not realistically possible, but maybe i'm wrong)